util-bukkit/src/main/i18n/templates/strings.properties

```diff
@@ -227,90 +236,90 @@ command.playerNotOnline = {0} is not currently online
 # {0} = a player
 command.playerLocationUnavailable = {0}'s current location is unavailable
 
-
 # {0} = the page
 # {1} = the total pages
-pageHeader = Page {0} of {1}
+command.pageHeader = Page {0} of {1}
```

Comment:
How about `misc.pageHeader` ? Could be used for GUIs

Reply:
I will move all 4 page strings into `misc` !